util/os.go

```diff
@@ -19,18 +19,16 @@ func SetOsProxy(port int) error {
 
 	network, err := getDefaultNetwork()
 	if err != nil {
-		return fmt.Errorf("failed to get network interfaces, stdout: %s: %w", string(network), err)
+		return fmt.Errorf("failed to get network interfaces: %w", err)
```

Comment:
I think it should be network or network service. We extract an interface from the `route` command and try to find a NS that uses it.

Reply:
I rewrited it as `failed to get default network`, cause we are calling `getDefaultNetwork`.